openpose/main.py

```diff
@@ -28,7 +28,11 @@ async def segment_video(
             temp_video_file.write(video_content)
             temp_video_file_path = temp_video_file.name
 
-            pose_data = perform_openpose_pose_estimation(temp_video_file_path, options)
+            multi_person_detection = options.get("multi_person_detection", False)
```

Comment:
Nice and clean solution to add an option here instead of adding a second endpoint! 👍🏼